### PR TITLE
Add release notes for changes to Account and Admin consoles

### PR DESCRIPTION
**Pull request by jonkoops**

### CODE REVIEW

File: docs/documentation/release_notes/topics/25_0_0.adoc [hunk 1]
```diff
@@ -1,3 +1,11 @@
+= Account Console v2 theme removed
+
+The Account Console v2 theme has been removed from {project_name}. The Account Console v2 theme was deprecated in {project_name} 24 and replaced by the Account Console v3 theme. If you are still using the Account Console v2 theme, you should migrate to the Account Console v3 theme.
```
Comment (andymunro) [inline docs/documentation/release_notes/topics/25_0_0.adoc:3]:
```suggestion
The Account Console v2 theme has been removed from {project_name}. This theme was deprecated in {project_name} 24 and replaced by the Account Console v3 theme. If you are still using this theme, you should migrate to the Account Console v3 theme.
```

**Review by andymunro — COMMENTED** [review]
Looks good @jonkoops with a few changes.

**Comment by jonkoops** [comment]
Done. @andymunro could you give it another pass?

**Review by andymunro — APPROVED** [review]
Looks perfect, @jonkoops

**Comment by jonkoops** [comment]
Had to push a fix for a link, but contents are otherwise the same. This can be merged.

**Review by ahus1 — APPROVED** [review]
Approving based on Andy's review.

**Pull request merged**